lib/srv/discovery/fetchers/kube_services.go

```diff
@@ -131,7 +131,7 @@ func (f *KubeAppFetcher) getServices(ctx context.Context) ([]v1.Service, error)
 			} else if match {
 				result = append(result, s)
 			} else {
-				f.Log.WithField("service_name", s.Name).Debug("Service doesn't match labels.")
+				f.Logger.DebugContext(ctx, "Service doesn't match labels", "service_name", s.Name)
```

Comment:
```suggestion
				f.Logger.DebugContext(ctx, "Service doesn't match labels", "service", s.Name)
```

To match other calls in this file.